pipewire.md: add a page on setting up pipewire (#540)
void-linux/void-docs: src/config/graphical-session/xorg.md

driver package.

Xorg defaults to DDX drivers if they are present, so in this case modesetting
must be explicitly selected: see
[Forcing the modesetting driver](#forcing-the-modesetting-driver).

For advanced configuration, see
[modesetting(4)](https://man.voidlinux.org/modesetting.4).

### Proprietary Drivers

Void also provides [proprietary NVIDIA drivers](./graphics-drivers/nvidia.md),
which are available in the

> [nonfree repository](../../xbps/repositories/index.md#nonfree).

Reviewer: These changes don't seem to be related to pipewire, is it a mistake?

Author: auto-formatting of my text editor :/ i'll revert this

The `xinit` package provides the [startx(1)](https://man.voidlinux.org/startx.1)
script as a frontend to [xinit(1)](https://man.voidlinux.org/xinit.1), which can
be used to start X sessions from the console. For example, to start
[i3(1)](https://man.voidlinux.org/i3.1), edit `~/.xinitrc` to contain `exec
/bin/i3` on the last line.

To start arbitrary programs together with an X session, add them in `~/.xinitrc`
before the last line. For example, to start
[pipewire(1)](https://man.voidlinux.org/pipewire.1) before starting i3, add
`pipewire &` before the last line.

> Here is the full example of `~/.xinitrc` which starts `pipewire` and `i3`:

Reviewer 2: ```suggestion
A `~/.xinitrc` file which starts `pipewire` and `i3` is shown below:
```

exec /bin/i3
```

To start arbitrary programs together with an X session, add them in `~/.xinitrc`

> before the last line. For example, to start

Reviewer 3: ` <...> before starting the window manager / DE.`
Makes the example code block is superfluous in my opinion (or could be integrated in the above code block even)

Author: True. I've tweaked this a bit.

> [pipewire(1)][https://man.voidlinux.org/pipewire] before starting i3, add the

Reviewer 2: ```suggestion
[pipewire(1)][https://man.voidlinux.org/pipewire.1] before starting i3, add the
```

following:

```
pipewire &
exec /bin/i3
```

> 

Reviewer 2: If possible, split this change into a separate commit.
